.circleci/config.yml

```diff
@@ -44,6 +44,7 @@ workflows:
       - docker-compose-ubuntu-cpp
       - docker-compose-centos-cpp
       - docker-compose-presto-native
+      - linux-join-fuzzer-run
```

Comment:
Would it make sense to move this job next to linux-aggregate-fuzzer-run?

Reply:
+1